docs/fundamentals/runtime-libraries/system-environment-getenvironmentvariable.md

```diff
@@ -29,29 +32,27 @@ On Windows systems, the environment block of the current process includes:
 
 If environment variables are created after the process has started, you can use this method to retrieve only those variables that were created by calling the <xref:System.Environment.SetEnvironmentVariable(System.String,System.String)> method or the <xref:System.Environment.SetEnvironmentVariable(System.String,System.String,System.EnvironmentVariableTarget)> method with a `target` value of .<xref:System.EnvironmentVariableTarget.Process?displayProperty=nameWithType>.
 
-### On non-Windows systems
+### On Unix-like systems
 
-On non-Windows systems, the environment block of the current process includes the following environment variables:
+On Unix-like systems, the environment block of the current process includes the following environment variables:
```

Comment:
Is it worth mentioning that macOS is "unix-like", or is that obvious?

Reply:
We use "Unix-like" throughout the docs, so I think this is okay.